ihmc-communication/src/main/java/us/ihmc/communication/crdt/CRDTInfo.java

```diff
@@ -6,7 +6,7 @@ public class CRDTInfo
 {
    private final ROS2ActorDesignation actorDesignation;
    private final int maxFreezeDuration;
-   private long updateNumber = -1; // Increment before first update
+   private long updateNumber = 0;
```

Comment:
This made it not frozen on the first tick, so never frozen until something calls freeze.